SharpRAT/Server/Server/SocketServer.cs

```diff
@@ -113,8 +113,18 @@ public static void ParseData(Client client, string data)
 
             if(data.StartsWith("<GET-TASKMGR-REG>"))
             {
-                data = data.Replace("<GET-TASKMGR-REG>", "");
-                client.bTaskmgrDisabled = Convert.ToBoolean(int.Parse(data));
+                try
+                {
+                    data = data.Replace("<GET-TASKMGR-REG>", "");
+                    if (data == "<ERROR-NOTFOUND>")
+                        Log.Error($"Task manager switch received error-notfound");
```

Comment:
Add brackets here, usually when we're not including the brackets we have only 1 line function under.